src/videojs.ima.js

```diff
@@ -1092,7 +1092,22 @@
       clearInterval(updateTimeIntervalHandle);
       clearInterval(seekCheckIntervalHandle);
       clearInterval(resizeCheckIntervalHandle);
-      player.one('play', player.ima.setUpPlayerIntervals_);
+      if(player.el()) player.one('play', player.ima.setUpPlayerIntervals_);
```

Comment:
@shawnbuso the purpose of that line is to avoid adding events to a disposed element, which is possible if we dispose the player before it completes initialization of the plugins and all handlers are fired